app/services/json_web_token.rb

```diff
@@ -0,0 +1,15 @@
+class JsonWebToken
+  HMAC_SECRET = Rails.application.secrets.secret_key_base
+  ALGORITHM_TYPE = 'HS256'.freeze
+
+  def self.encode(user_id)
+    exp = 24.hours.from_now.to_i
+    payload = { user_id:, exp: }
+    JWT.encode payload, HMAC_SECRET, ALGORITHM_TYPE
+  end
+
+  def self.decode(token)
+    decoded = JWT.decode(token, SECRET_KEY)[0]
+    HashWithIndifferentAccess.new decoded
+  end
+end
```

Comment:
-  [ ] Please fix this issue to test other endpoints that have been implemented.

![image](https://user-images.githubusercontent.com/55635977/193369977-f2a4f9e9-3427-4140-9e2a-f015b874ddad.png)

![image](https://user-images.githubusercontent.com/55635977/193370059-cab81c59-6e01-430e-b05d-dfe0e387f1b9.png)